javascript-and-json/main.js

```diff
@@ -0,0 +1,33 @@
+var arr = [
+  {
+    isbn: 123,
+    title: 'Title A',
+    author: 'Author A'
+  },
+
+  {
+    isbn: 124,
+    title: 'Title B',
+    author: 'Author B'
+  },
+
+  {
+    isbn: 125,
+    title: 'Title C',
+    author: 'Author C'
+  }
+];
+console.log('arr:', arr);
+console.log('typeof arr:', typeof arr);
+
+var arrJSON = JSON.stringify(arr);
+console.log('arrJSON:', arrJSON);
+console.log('typeof arrJSON:', typeof arrJSON);
+
+var String = JSON.stringify({ 'Number id': 12, 'String name': 'toad' });
```

Comment:
You shouldn't be using the stringify method for this step, I want you to write out the JSON string by hand so that it can be parsed by JavaScript into an object.

![image](https://user-images.githubusercontent.com/32880187/132047193-2a96ae89-1119-45df-b9db-52782a6484ce.png)

Also, The `Number` and `String` portion is just meant to tell you what data type that property is meant to hold, it isn't meant to be a part of the property name.